build-managed.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 working_tree_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-$working_tree_root/run.sh build-managed $*
```

Comment:
I've run into this issue as well.  This works on Windows but is broken on Linux:

`msbuild.sh /t:RebuildAndTest /p:XunitOptions="-showprogress -parallel none"`